src/app/components/infinite-scroll/index.html

```diff
@@ -0,0 +1,29 @@
+<sky-demo-page
+  pageTitle="Infinite scroll">
+
+  <sky-demo-page-summary>
+    The infinite scroll component allows users to load data as the page scrolls.
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="hasMore"
+      defaultValue="false"
+      isOptional="true"
+    >
+    A boolean specifying whether there is more data to load. If this is false, infinite scroll will hide and stop calling the load function.
```

Comment:
I'd change the first sentence to "Indicates whether more data is available to load."

I'd change "this is" to "set to" and format "false" with "stache-code" tags. I'd also change "will hide and stop" to "hides and stops."